net/quagga/src/opnsense/mvc/app/controllers/OPNsense/Quagga/Api/ServiceController.php

```diff
@@ -139,9 +149,10 @@ public function reconfigureAction()
                 $this->startAction();
             }
 
-            return array("status" => "ok");
-        } else {
-            return array("status" => "failed");
+            return array('status' => 'ok');
+        }
```

Comment:
btw: is this perl? :D

the style fixer will revert this likely

Reply:
@fichtner I don't write code in perl because others should understand the code I write ;)
One of my friends at university does not like it if I write code with lots of regular expressions and implicit variables (he says hard to read).
Maybe the reason he does not like ruby...

Reply:
@fichtner why should I start it after quagga? When starting, the packets are already sent (no delay). The filter rule needs to be in place already

Reply:
not much damage if that happens.

Reply:
filter seems to be last in multiple scripts, I don't want to break with tradition :)

https://github.com/opnsense/core/blob/master/src/etc/rc.reload_all
https://github.com/opnsense/core/blob/master/src/etc/rc.newwanip
etc

Reply:
for perl I just meant the bracing:

}
else {

doesn't really add any value and is against our PSR I think.

Reply:
its against PSR2 but not against PSR1 - I have reverted that